integration-tests/src/test/java/io/quarkiverse/githubaction/it/RepositoryActionTest.java

```diff
@@ -44,7 +44,9 @@ public void shouldPrintRepositoryNameWhenRunningOnGitHub(LaunchResult result) {
     @Launch(value = {}, exitCode = 1)
     @EnabledIf("isLocal")
     public void shouldFailWhenRunningLocally(LaunchResult result) {
-        assertThat(result.getErrorOutput()).contains(REPOSITORY_NAME);
+        String fullOutput = result.getOutput() + "\n" + result.getErrorOutput();
```

Comment:
I don't know exactly why but in my case the ERROR log was in `result.getOutput()`. I merged both so that we are safe. We don't really care about that.